src/const/permissions.ts

```diff
@@ -5,13 +5,14 @@ const permissions = {
   canSeeUsers: 'can_see_users',
   canSeeRoles: 'can_see_roles',
   canSeeApplications: 'can_see_applications',
+  canCreateApplications: 'can_create_applications',
   canCreateForms: 'can_create_forms',
   canCreateResources: 'can_create_resources',
   canManageResources: 'can_manage_resources',
   canManageForms: 'can_manage_forms',
   canManageApplications: 'can_manage_applications',
   canManageApiConfigurations: 'can_manage_api_configurations',
-  canCreateApplications: 'can_create_applications',
+  canManageTemplates: 'can_manage_templates',
```

Comment:
@matheus-relief I think this should be a permission at application level, but it is not visible in apps I think
the can_manage_applications should allow to edit templates, and this new permission should only enable selected roles per app to do so in the related apps